rootfs.sh

```diff
@@ -76,12 +76,12 @@ mkfs.ext4 -L root ${root_dev}
 echo Mounting root filesystem
 mount ${root_dev} ${root_mnt}
 
-echo Unpacking linaro tarball
-tar xfzp ${linaro_tarball} -C ${root_mnt} --strip-components 1
+echo Unpacking Ubuntu tarball
+tar xfzp ${ubuntu_tarball} -C ${root_mnt} --strip-components 0
 
-echo Renaming linaro user to parallella
-[ -d ${top}/mnt/rootfs/home/parallella ] || mv ${top}/mnt/rootfs/home/linaro ${top}/mnt/rootfs/home/parallella
-sed -i 's/linaro/parallella/g' ${top}/mnt/rootfs/etc/{group,shadow,passwd}
+#echo Renaming linaro user to parallella
```

Comment:
Is there a 'ubuntu' user in the ubuntu base image? If yes, rename that user. If no, remove these lines instead of just commenting them out.

Reply:
I don't think there's a 'ubuntu' user in the base image.
https://wiki.ubuntu.com/Base/InstallationExample